cranelift-codegen/meta/src/cdsl/instructions.rs

```diff
@@ -1081,9 +1081,9 @@ fn bind(
 fn bind_vector(
     inst: Instruction,
     lane_type: LaneType,
-    num_lanes: u64,
     mut value_types: Vec<ValueTypeOrAny>,
 ) -> BoundInstruction {
+    let num_lanes = 128 / lane_type.lane_bits();
```

Comment:
Well, I was doing the `128 / lane_bits` calculation all over the place so at least now it is in one place. I could replace `128` with a static variable?

Reply:
I guess, it was one thing when it was in the x86 code, close to the places that will need changes anyway when we do AVX2. Here, it's in platform-independent code, so it feels more out of place.

Would it help to add a `num_lanes` function, next to `lane_bits`, where you could pass in 128 and it returns that width divided by the `lane_bits()` value?

Alternatively, maybe we could keep the code as you have it here, and just rename `bind_vector` to `bind_vector128` or so, so that it's clear on the caller side that this is for 128-bit vectors. Does that make sense?

Reply:
I decided for a variation on the first option: I kept the calculation where it is but added a parameter `vector_size_in_bits` to `bind_vector_from_lane` which should future-proof this. That change is really a part of #855 (a smaller PR than this constant-related PR) so I will add a comment there about this.